proto/account/query.proto

```diff
@@ -4,12 +4,69 @@ package tendermint.spn.account;
 import "google/api/annotations.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
 // this line is used by starport scaffolding # 1
+import "account/coordinator_by_address.proto";
+import "account/coordinator.proto";
 
 option go_package = "github.com/tendermint/spn/x/account/types";
 
 // Query defines the gRPC querier service.
 service Query {
     // this line is used by starport scaffolding # 2
+
+	// Queries a coordinatorByAddress by index.
+	rpc CoordinatorByAddress(QueryGetCoordinatorByAddressRequest) returns (QueryGetCoordinatorByAddressResponse) {
+		option (google.api.http).get = "/tendermint/spn/account/coordinatorByAddress/{address}";
+	}
+
+	// Queries a list of coordinatorByAddress items.
+	rpc CoordinatorByAddressAll(QueryAllCoordinatorByAddressRequest) returns (QueryAllCoordinatorByAddressResponse) {
```

Comment:
I think we can manually remove this query with everything related to it.

The benefit of this `coordinatorByAddress` structure is to get the coordinator from the address but if we want to get all of them, we can use the query generated for `coordinator`